opencti-platform/opencti-front/src/components/FilterIconButtonContentWithRedirectionAndDeprecatedInstancesSearch.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FunctionComponent } from 'react';
+import { Link } from 'react-router-dom';
+import { graphql, useLazyLoadQuery } from 'react-relay';
+import {
+  FilterIconButtonContentWithRedirectionAndDeprecatedInstancesSearchQuery,
+} from './__generated__/FilterIconButtonContentWithRedirectionAndDeprecatedInstancesSearchQuery.graphql';
+
+export const filterIconButtonContentWithRedirectionAndDeprecatedInstancesSearchQuery = graphql`
+    query FilterIconButtonContentWithRedirectionAndDeprecatedInstancesSearchQuery(
+        $id: String!
+    ) {
+        stixObjectOrStixRelationship(id: $id) {
+            ... on StixCoreObject {
+                id
+            }
+        }
+    }
+`;
+
+interface FilterIconButtonContentWithRedirectionAndDeprecatedInstancesSearchProps {
+  filterId: string,
+  displayedValue: string,
+}
+
+const FilterIconButtonContentWithRedirectionAndDeprecatedInstancesSearch: FunctionComponent<FilterIconButtonContentWithRedirectionAndDeprecatedInstancesSearchProps> = ({
```

Comment:
😱 Can you find a more concise naming for this file ?

Reply:
done :D